src/blockchainTx/web3/ContractsService.ts

```diff
@@ -0,0 +1,215 @@
+            data: hashed,
+            uri: '',
+        };
+        const attestation_2 = {
+            claimType: claimType_2,
+            scheme: 1,
+            issuer:  config.ownerAddress,
+            signature: signed2.signature,
+            data: hashed2,
+            uri: '',
+        };
+        this.ecosystemContract.methods.createNewCentralizedUser(
+            newUser.userAddress,
+            [ attestation_1.claimType, attestation_2.claimType ],
+            [ attestation_1.issuer, attestation_2.issuer ],
+            attestation_1.signature + attestation_2.signature.slice(2),
+            attestation_1.data + attestation_2.data.slice(2),
+            [32, 32]).send({from: config.ownerAddress,
+            gas: 3000000,
+            gasPrice: '1'})
+            .then(function(receipt){
+                console.log(receipt);
+            });
+    }
+
+    async predictIdentityAddress(wallet) {
+        const nonce = await new Promise(resolve => {
+            this.web3.eth.getTransactionCount(wallet, (err, count) => {
+                resolve(count);
+            });
+        });
+        const address =
+            '0x' + this.web3.utils.sha3(RLP.encode([wallet, nonce])).substring(26, 66);
+        return address.toString();
+    }
+
+    public updateScreener(svandisData: UpdateScreenerDto) {
+        const dataAddress = svandisData.dataAddress;
+        const newData = svandisData.newDataLoad;
+        const consensusUsers = [config.ownerAddress];
+        const consensusUserNewRatings = [1];
+        const metConsensus = [true];
+        this.ecosystemContract.methods.updateSvandisData (dataAddress,
+            newData,
+            consensusUsers,
+            consensusUserNewRatings,
+            metConsensus).send({from: config.ownerAddress,
+            gas: 3000000,
+            gasPrice: '1'})
+            .then(function(receipt){
+                console.log(receipt);
+            });
+    }
+
+    public removeUser(user: UserRemovalDto) {
+        this.ecosystemContract.methods.removeUser (user.userAddressForRemoval,
+        ).send({from: config.ownerAddress,
+            gas: 1000000,
+            gasPrice: '1'})
+            .then(function(receipt){
+                console.log(receipt);
+            });
+    }
+
+    public swapCentralizedUserRecovery(user: SwapRecoveryCentralizedDto) {
+        this.ecosystemContract.methods.swapMainKeyForSvandisCentralizedUserAccounts (
+            user.currentAddress,
+            user.newRecoveryAddress).send({from: config.ownerAddress,
+            gas: 2000000,
+            gasPrice: '1'})
+            .then(function(receipt){
+                console.log(receipt);
+            });
+    }
+
+    public addExtraKeyForSvandisCentralizedUserAccounts(user: AddExtraRecoveryCentralizedDto) {
+        this.ecosystemContract.methods.addExtraKeyForSvandisCentralizedUserAccounts (
+            user.currentAddress, user.newRecoveryMethod).send({from: config.ownerAddress,
```

Comment:
Let's use consistent method call style. In some places you have whitespace between method's name and parentheses, `.this` separated with a dot by newline. So let's chain method calls with each new method on a separate line and not put whitespace between method's name and parentheses.

Reply:
Yes this turned out inconsistent. Revising this now